recipes/rauth/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install 
```

Comment:
Please remove the `bld.bat` and `build.sh` in favor of the `script` entry in the `meta.yaml`:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L29